gulpfile.js

```diff
@@ -142,4 +153,34 @@ gulp.task('watch', function() {
 	gulp.watch('**/*.css', ['lint:css']);
 });
 
+// Server code from http://blog.overzealous.com/post/74121048393/why-you-shouldnt-create-a-gulp-plugin-or-how-to
+gulp.task('server', ['watch', 'build:extensions'], function(callback) {
```

Comment:
Dependency on `watch` is unnecessary, since we're registering our own watch tasks inside `server`.